RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -39,7 +39,7 @@ struct ManageSubscriptionsView: View {
     @Environment(\.navigationOptions)
     var navigationOptions
 
-    @StateObject
+    @ObservedObject
```

Comment:
This is the correct approach. `@StateObject` should only be used when the view creates the object. If it's injected (as in this case), then it should be `@ObservedObject` 👍

Reply:
I've rewritten it using `Binding` + `onChangeOf` 🚀